src/Pim/Bundle/CatalogBundle/tests/integration/ElasticSearch/PimCatalogTextIndexConfigurationIntegration.php

```diff
@@ -0,0 +1,580 @@
+<?php
+
+namespace Pim\Bundle\CatalogBundle\tests\integration\ElasticSearch;
+
+use Elasticsearch\ClientBuilder;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+/**
+ * This integration tests checks that given an index configuration and some products indexed
+ * the text research are consistent.
+ *
+ * @author    Samir Boulil <samir.boulil@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class PimCatalogTextIndexConfigurationIntegration extends KernelTestCase
```

Comment:
I would prefer that this class is totally independent from Symfony. Unless you need it to be?